docs/custom_framework.md

```diff
@@ -1,39 +1,39 @@
 ## 🎎 Custom Framework Config
 
-If you are using an i18n framework that does not have built-in support by this extension or you are using a custom i18n implementation, you can always make this extension support for your framework.
+If you are using an i18n framework that does not have built-in support by this extension, or you are using a custom i18n implementation, you can always make this extension support for your framework.
 
 Create file `.vscode/i18n-ally-custom-framework.yml` then copy and paste the configs below.
 
 ```yaml
 # .vscode/i18n-ally-custom-framework.yml
 
-# An array of string which contains Language Ids defined by vscode
+# An array of strings which contain Language Ids defined by VS Code
```

Comment:
```suggestion
# An array of strings which contain Language Ids defined by VSCode
```

Reply:
The official website refers to it as "VS Code" 7 times.

Reply:
emmm..., you are right. Never mind about this, I will replace all the VSCode to VS Code after merging.